api/cron/compute-signals.js

```
@@ -0,0 +1,731 @@
+// api/cron/compute-signals.js — 서버 사전 계산 시그널 cron
+// Vercel Cron이 10분마다 호출: */10 * * * *
+// ⚠️ src/engine/ ESM import 불가 (Edge runtime 호환) — 로직 인라인 복사
+// ⚠️ 수정 시 src/engine/taCalculator.js / src/engine/compositeScorer.js 와 동기화 필수
+import { getSnap, setSnap, recordCronFailure } from '../_price-cache.js';
```

ESLint

[error] 'getSnap' is defined but never used. Allowed unused vars must match /^[A-Z_]/u.

(no-unused-vars)